.github/workflows/stale.yml

```diff
@@ -0,0 +1,83 @@
+# See https://github.com/actions/stale
+
+name: Stale issues and pull-requests
+
+on:
+  schedule:
+    # Run once a day
+    # N.B. "should" be quoted, according to
+    # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onschedule
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    if: "github.repository == 'SciTools/python-stratify'"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          # Idle number of days before marking issues/prs stale.
+          days-before-stale: 500
```

Comment:
@HGWright Just wondering whether we could have an environment variable for this which can then be used here and in the text for the `stale-issue-message`.

Also applies to the `days-before-close` e.g.,

```yaml
- env:
    DAYS_BEFORE_STALE: 500
    DAYS_BEFORE_CLOSE: 28

...

    days-before-stale: ${{ env.DAYS_BEFORE_STALE }}
    days-before-close: ${{ env.DAYS_BEFORE_CLOSE }}

    stale-issue-message: |
      In order ... as stale after ${{ env.DAYS_BEFORE_STALE }} days of inactivity.

      ...

      Otherwise ... in ${{ env.DAYS_BEFORE_CLOSE }} days time.

```

Would that work here?
```

Reply:
Thanks @bjlittle. Like the idea as it would make changing that length of time easier as it would only need to change in one place.